docs/walk-through/quick-start.md

```diff
@@ -1,60 +1,47 @@
 # Quick Start
 
-## Install in GitHub Codespaces
+## Install Hera
 
-You can install Argo and try running Hera code in GitHub Codespaces. First, open the Hera repository in Codespaces using the link below:
-
-[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/argoproj-labs/hera)
+[![Pypi](https://img.shields.io/pypi/v/hera.svg)](https://pypi.python.org/pypi/hera)
 
-Then, you can run the `make` commands to install k3d (a lightweight Kubernetes CLI), and then install the Argo Workflows controller on the cluster:
+Hera is available on PyPi as the `hera` package. Add this dependency to your project with your favourite package
+manager, e.g. [pip](https://packaging.python.org/en/latest/guides/tool-recommendations/#installing-packages) or [poetry](https://python-poetry.org/), or install directly with
```

Comment:
I'm not sure the intent here -- pip is listed as a package manager, which I don't think it is, and then it's listed again under "or install directly". Perhaps this instead?

```suggestion
Hera is available on PyPi as the `hera` package. Add this dependency to your project with your favourite package
manager, e.g. [setuptools](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-and-requirements), [uv](https://docs.astral.sh/uv/concepts/projects/dependencies/) or [poetry](https://python-poetry.org/), or install directly with
```